src/Utility.ts

```diff
@@ -48,20 +50,21 @@ export namespace Utility {
         }
     }
 
-    export function getRestartConfig(): boolean {
-        const config: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration('tomcat');
-        if (config) {
-            return config.get<boolean>(Constants.RESTART_CONFIG_ID);
+    export function browse(localhost: boolean, port: string, subPath?: string, url?: string): void {
+        if (localhost) {
+            // tslint:disable-next-line:no-http-string
+            opn(`http://localhost:${port}/${subPath || ''}`);
+        } else {
+            opn(url);
```

Comment:
The behavior is confusing. If isLocalhost is set to false, then `url` is used without using port and subpath. This function seems redundant because what it does is just to form a url for localhost. Suggest removing this function.

Reply:
will remove the confusing the utility method